routes/deductions/deductions.controller.js

```diff
@@ -208,3 +282,15 @@ module.exports = function(app) {
     doRedirect,
   )
 }
+
+const seniorRedirect = () => {
+  return (req, res, next) => {
+    const dobInDays = getDateDelta(req.session.personal.dateOfBirth)
+    if (process.env.NODE_ENV === 'production') {
+      if (dobInDays <= SIXTYFIVE_YEARS_INDAYS) {
+        return res.redirect('/trillium/rent')
+      }
+    }
+    return next()
+  }
+}
```

Comment:
We don't need the "return a function from a function" pattern here because we're not passing in anything to `seniorRedirect`.

We can rewrite this more simply as 

```
const seniorRedirect = (req, res, next) => {
    const dobInDays = getDateDelta(req.session.personal.dateOfBirth)
    if (process.env.NODE_ENV === 'production') {
      if (dobInDays <= SIXTYFIVE_YEARS_INDAYS) {
        return res.redirect('/trillium/rent')
      }
    }
    return next()
  }
```

And then we can call it on line 149, 155, etc as 

```
   seniorRedirect,
```

(without the brackets)